.gitignore

```diff
@@ -133,3 +133,4 @@ docs/pages/versions/*/react-native/*.diff
 /packages/**/*.xcframework
 /packages/**/*.spec.json
 /packages/**/Info-generated.plist
+/packages/**/plugin/build/
```

Comment:
Sorry, I misspoke -- we want to commit (not .gitignore) the built plugin JS but want GitHub to collapse the built JS files during code review. This is handled in `.gitattributes`.